starter-code/css/main.css

```diff
@@ -0,0 +1,126 @@
+body {
+  border-left:5px #ccc solid;
+  margin:0 0 0 5px;
+  font-family: Lato;
+}
+
+header {
+  max-width:700px;
+  padding-left:24px;
+}
+
+article {
+  max-width:700px;
+  padding-left:24px;
+}
+
+footer {
+  padding-left:24px;
+  max-width:700px;
+  border-top:1px #ccc solid;
+  padding-top:25px;
+}
+article h3{
+  color:#ccc;
+  font-weight:100;
+}
+
+img {
+  max-width: 100%;
+  max-height: 100%;
+}
+
+h1 {
+  -webkit-margin-before: 0;
+  -webkit-margin-after: 0;
+  -webkit-margin-start: 0px;
+  font-weight:lighter;
```

Comment:
Rather than using the value "lighter" you should specify the font-weight given by the google font. i.e. 300.